about.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <link href="fonts/cocogoose/cocogoose.css" rel="stylesheet">
+  <link rel="stylesheet" href="./css/global.css">
+  <link rel="stylesheet" href="./css/about.css">
+  <script src="./script.js"></script>
+</head>
+
+<body>
+  <div id="mobile-menu">
+    <div class="close-btn">
+      <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-x"><line x1="18" y1="6" x2="6" y2="18"></line><line x1="6" y1="6" x2="18" y2="18"></line></svg>
+    </div>
+    <ul>
+      <li><a href="/">Home</a></li>
+      <li><a href="/about.html">About</a></li>
```

Comment:
**Requirements for your About Page:**

- It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
- It contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.
- It contains a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them.

**_Kindly provide the requested style on your Template._**